src/cmd/cli/command/compose.go

```diff
@@ -64,6 +99,30 @@ func makeComposeUpCmd() *cobra.Command {
 			if err != nil {
 				return err
 			}
+
+			project, err := loader.LoadProject(cmd.Context())
+			if err != nil {
+				return err
+			}
+
+			managedServices, unmanagedServices := splitManagedAndUnmanagedServices(project.Services)
+
+			var hasManagedServices bool
+			if hasManagedServices, err = canUseManagedServices(managedServices); err != nil {
+				return err
+			}
+
+			if hasManagedServices {
+				term.Warnf("Defang cannot monitor status of the following managed service(s): %v.\n   To check if the managed service is up, check the status of the service which depends on it.", managedServices)
+			}
```

Comment:
```suggestion
			var canUseManagedSvcs bool
			if canUseManagedSvcs, err = canUseManagedServices(managedServices); err != nil {
				return err
			}

			if len(managedServices) > 0 && canUseManagedSvcs {
				term.Warnf("Defang cannot monitor status of the following managed service(s): %v.\n   To check if the managed service is up, check the status of the service which depends on it.", managedServices)
			}
```

i assume this is called this because right now `canUse` == `has`, but leaking that out into the caller feels like its ripe for misunderstanding.